internal/cmd/clone/clone.go

```diff
@@ -3,12 +3,36 @@
 package clone
 
 import (
+	"strings"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
 	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/gittuf/gittuf/internal/tuf"
 	"github.com/spf13/cobra"
 )
 
 type options struct {
-	branch string
+	branch           string
+	expectedRootKeys PathList
+}
+
+// PathList is a custom type to represent a list of paths
+type PathList []string
+
+// String implements part of the pflag.Value interface.
+func (p *PathList) String() string {
```

Comment:
```suggestion
func (p PathList) String() string {
```

Since PathList is a slice, this doesn't need to be a pointer method to match the interface, if I'm not mistaken.

Reply:
We need it for the append values to propogate